README.md

```diff
@@ -1,19 +1,218 @@
+list the repositories for Allstar to be enabled on in your organization like so:
+
+```
+optConfig:
+  optInRepos:
+  - repo-one
+  - repo-two
+```
+
+To switch to the opt-out strategy, set that option to true:
+
+```
+optConfig:
+  optOutStrategy: true
+```
+
+If you with to enable Allstar on all but a few repositories, you may use opt-out
+and list the repositories to disable:
+
+```
+optConfig:
+  optOutStrategy: true
+  optOutRepos:
+  - repo-one
+  - repo-two
+```
+
+### Repository Override
+
+Individual repositories can also opt in or out using configuration files inside
+those repositories. For example, if the organization is configured with the
+opt-out strategy, a repository may opt itself out by including the file
+`.allstar/allstar.yaml` with the contents:
+
+```
+optConfig:
+  optOut: true
+```
+
+Conversely, this allows repositories to opt-in and enable Allstar when the
+organization is configured with the opt-in strategy. Because opt-in is the
+default strategy, this is how Allstar works if the `.allstar` repository doesn't
+exist.
+
+In the organization-level `allstar.yaml`, repository override may be disabled with the setting:
+
+```
+optConfig:
+  disableRepoOverride: true
+```
+
+This allows an organization-owner to have a central point of approval for
+repositories to request an opt-out through a GitHub PR. Understandably, Allstar
+or individual policies may not make sense for all repositories.
+
+### Policy Enable
+
+Each individual policy configuration file (see below) also contains the exact
+same `optConfig` configuration object. This allows granularity to enable
+policies on individual repositories. A policy will not take action unless both
+it is enabled and Allstar is enabled as a whole.
+
+### Definition
+
+https://pkg.go.dev/github.com/ossf/allstar@v0.0.0-20210728182754-005854d69ba7/pkg/config#OrgOptConfig
+https://pkg.go.dev/github.com/ossf/allstar@v0.0.0-20210728182754-005854d69ba7/pkg/config#RepoOptConfig
+
+## **Actions**
+
+Each policy can be configured with an action that Allstar will take when it
+detects a repository to be out of compliance.
+
+- `log`: This is the default action, and actually takes place for all
+  actions. All policy run results and details are logged. Logs are currently
+  only visible to the app operator, plans to expose these are under discussion.
+- `issue`: This action creates a GitHub issue. Only one issue is created per
+  policy, and the text describes the details of the policy violation. If the
+  issue is already open, it is pinged with a comment every 24 hours (not
+  currently user configurable). Once the violation is addressed, the issue
+  should be closed by Allstar within 5-10 minutes.
+- `fix`: This action is policy specific. The policy will make the changes to the
+  GitHub settings to correct the policy violation. Not all policies will be able
+  to support this (see below).
+
+Proposed, not yet implemented, actions. Further discussion and definition needed.
```

Comment:
s/, not yet/, but not yet
remove , between "implemented, actions"

Reply:
definitions will be added in the future

Reply:
done